packages/framework/src/Resources/views/Admin/Form/theme.html.twig

```diff
@@ -1083,9 +1083,9 @@
 
 {% macro icon(attr) %}
     {% if attr.icon is defined %}
-        <i class="{{ attr.iconClass|default('') }} svg svg-{{ attr.iconType|default('info') }} in-icon cursor-help form-line__item--info js-tooltip"
-           data-toggle="tooltip" data-placement="{{ attr.iconPlacement|default('bottom') }}"
-           title="{{ attr.iconTitle|default('') }}"
-        ></i>
+        {{ info_icon(attr.iconTitle|default(''),
+            tooltipPlacement: attr.iconPlacement|default('bottom'),
+            additionalClasses: 'form-line__item--info' ~ attr.iconClass|default('')
```

Comment:
when `iconClass` is set, it is appended to "form-line__item--info" with no space between it - you can see it e.g. in the category form type - `automatedFilters` field renders the SVG classes as `...form-line__item--infomargin-left-10`

Reply:
nice catch! fixed in b346f09820